openapi/components/schemas/LtftApplicationAdminSummary.yml

```diff
@@ -0,0 +1,41 @@
+type: object
+readOnly: true
+properties:
+  id:
+    type: string
+    format: uuid
+    example: fc13458c-5b0b-442f-8907-6f9af8fc0ffb
+  personalDetails:
+    allOf:
+      - $ref: ./PersonalDetailsName.yml
+      - $ref: ./PersonalDetailsRegistration.yml
+  programmeName:
+    type: string
+    example: General Practice
+  proposedStartDate:
+    type: string
+    format: date
+  submissionDate:
+    type: string
+    format: date
+  daysToStart:
+    type: number
+  shortNotice:
+    type: boolean
+  tpd:
+    type: object
+    required:
```

Comment:
We don't want to put the TPD name in this as well?

Reply:
It wasn't on the ticket, so I've not added it... but possibly? 😅